workshop/SCHEDULE.md

```diff
@@ -3,15 +3,49 @@ title: Virtual Workshop Schedule
 nav_title: Schedule
 ---
 
-<!--See an example from a past virtual workshop here: https://github.com/AlexsLemonade/2020-may-training/wiki/Schedule --> 
+A preliminary schedule for the September 2022 Data Lab Virtual Training Workshop appears below.
 
-| Time        | Topic                                          |
-|-------------|------------------------------------------------|
-| **Day 1**   | **Date** <br> [Module]()                      |
-| 12:00 PM    | Welcome, Introductions and Getting Started     <br>[Welcome Slides (PDF)](../slides/ Workshop_Introduction.pdf)|
-| 5:00        | End             |
-| **Day 2**   | **Date**  | 
-| **Day 3**   | **Date**  |               
-| **Day 4**   | **Date**  | 
-| **Day 5**   | **Date**  |     
-| 5:00        | Adjourn   |
+⚠️ Not all links are working yet!
+
+*Note: All times are [EDT (UTC−04:00)](https://www.timeanddate.com/time/zones/edt).*
+
+
+| Time        | Topic                             | Location |
+|-------------|--------------------------------------------|----------------|
+| **Day 1**   | **2022-09-19** <br> [_**Introduction to R and the Tidyverse**_](https://github.com/AlexsLemonade/training-modules/tree/master/intro-to-R-tidyverse)                      |
+| 12:00 PM    | Welcome, Introductions and Getting Started  <br> [Welcome slides (PDF)](../slides/Workshop-Introduction.pdf)   | Zoom: [Main Session](../virtual-setup/zoom-procedures.md#joining-a-zoom-call) |
+ 1:00 PM     | [Introduction to R Slides (PDF)](../slides/Intro-R-RStudio.pdf) <br> [Introduction to base R](https://htmlpreview.github.io/?https://github.com/AlexsLemonade/training-modules/blob/{{site.release_tag}}/intro-to-R-tidyverse/01-intro_to_base_R.nb.html) | Zoom: [Main Session](../virtual-setup/zoom-procedures.md#joining-a-zoom-call)|
```

Comment:
This is an example of removing links from the table. I am eliminating all pdf links completely, as well as any links to `.nb.html` files.

Note that you will need to remove both the link itself, as well as the parenthesis around it and the brackets around the text that would be linked.


```suggestion
| 12:00 PM    | Welcome, Introductions and Getting Started  | Zoom: [Main Session](../virtual-setup/zoom-procedures.md#joining-a-zoom-call) |
 1:00 PM     | Introduction to base R | Zoom: [Main Session](../virtual-setup/zoom-procedures.md#joining-a-zoom-call)|
```